examples/twitter-compose-with-typeahead/src/Autocomplete.tsx

```diff
@@ -0,0 +1,287 @@
+                                  ]
+                                    .filter(Boolean)
+                                    .join(' ')}
+                                >
+                                  <AccountItem hit={item} />
+                                </div>
+                              </li>
+                            );
+                          })}
+                        </ul>
+                      )}
+                    </div>
+                  );
+                })}
+            </div>
+          </div>
+        </div>
+        <div className="box-footer">
+          <button type="submit" className="tweet-button">
+            Tweet
+          </button>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+type AccountItemProps = {
+  hit: Hit<Account>;
+};
+
+function AccountItem({ hit }: AccountItemProps) {
+  return (
+    <>
+      {(hit.follows_me || hit.followed_by_me) && (
+        <div className="account-follow-status">
+          <svg
+            className="account-follow-status-icon"
+            viewBox="0 0 20 20"
+            fill="currentColor"
+          >
+            <path
+              fillRule="evenodd"
+              d="M10 9a3 3 0 100-6 3 3 0 000 6zm-7 9a7 7 0 1114 0H3z"
+              clipRule="evenodd"
+            />
+          </svg>
+          {hit.follows_me && hit.followed_by_me && (
+            <span>You follow each other</span>
+          )}
+          {!hit.follows_me && hit.followed_by_me && (
+            <span>You follow them</span>
+          )}
+          {hit.follows_me && !hit.followed_by_me && (
+            <span>They follow you</span>
+          )}
+        </div>
+      )}
+      <div className="account-body">
+        <div className="account-avatar">
+          <img src={hit.image} alt="" />
+        </div>
+        <div>
+          <div className="account-name">
+            {hit.name !== '' && (
+              <span>
+                <Highlight hit={hit} attribute="name" />
+              </span>
+            )}
+            {hit.verified && (
+              <svg
+                className="account-icon"
+                viewBox="0 0 20 20"
+                fill="currentColor"
+                aria-label="Verified account"
+              >
+                <path
+                  fillRule="evenodd"
+                  d="M6.267 3.455a3.066 3.066 0 001.745-.723 3.066 3.066 0 013.976 0 3.066 3.066 0 001.745.723 3.066 3.066 0 012.812 2.812c.051.643.304 1.254.723 1.745a3.066 3.066 0 010 3.976 3.066 3.066 0 00-.723 1.745 3.066 3.066 0 01-2.812 2.812 3.066 3.066 0 00-1.745.723 3.066 3.066 0 01-3.976 0 3.066 3.066 0 00-1.745-.723 3.066 3.066 0 01-2.812-2.812 3.066 3.066 0 00-.723-1.745 3.066 3.066 0 010-3.976 3.066 3.066 0 00.723-1.745 3.066 3.066 0 012.812-2.812zm7.44 5.252a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z"
+                  clipRule="evenodd"
+                />
+              </svg>
+            )}
```

Comment:
Same for this.

Reply:
This is used in custom ranking, so this is helpful to understand relevance. We can move it into a component to streamline `Autocomplete`, wdyt?

Reply:
Same as above.

Reply:
Handled in 094b7bc.